icons/moinmoin.svg

```diff
@@ -0,0 +1 @@
0 0 .116.486l1.886 1.178-1.123.7a2.628 2.628 0 0 0-1.334-.38c-.454 0-.876.127-1.25.326l-1.152-.716 1.998-1.246a.355.355 0 0 0 .115-.487.137.137 0 0 0-.002 0l-1.25-2.006-.002-.002a.356.356 0 0 0-.332-.168.137.137 0 0 0-.002 0zm-.08.842.88 1.408-2.048 1.276-.877-1.41zm4.078.137L16.2 13.36l-.879 1.409-2.045-1.274zM8.906 15.4l1.668 1.04a.137.137 0 0 0 .002.001.355.355 0 0 0 .38-.01 1.963 1.963 0 0 1 1.075-.316c.428 0 .817.136 1.14.367a.356.356 0 0 0 .395.014.137.137 0 0 0 0-.002L15.32 15.4l.88 1.409-2.052 1.279v-.002a.354.354 0 0 0-.164.256 1.969 1.969 0 0 1-3.916-.094.355.355 0 0 0-.168-.272l-.002-.002-1.869-1.166z" style="fill:#000000;fill-opacity:0.941176;fill-rule:nonzero;stroke:none;stroke-width:0;stroke-dasharray:none"/><path d="M6.757 6.424c-1.38 0-2.5 1.113-2.5 2.483 0 .779.362 1.473.927 1.929v2.383a2.471 2.471 0 0 0-.927 1.929c0 1.37 1.12 2.483 2.5 2.483a2.494 2.494 0 0 0 2.502-2.483 2.48 2.48 0 0 0-1.139-2.082v-1.752l1.444.898a2.496 2.496 0 0 0 2.493 2.28c1.31 0 2.384-1 2.492-2.272l1.41-.875v1.74a2.479 2.479 0 0 0-1.104 2.063 2.494 2.494 0 0 0 2.503 2.483c1.38 0 2.5-1.113 2.5-2.483 0-.795-.378-1.504-.964-1.959v-2.325a2.471 2.471 0 0 0 .964-1.957c0-1.37-1.12-2.483-2.5-2.483A2.495 2.495 0 0 0 14.88 8.56l-1.864 1.158a2.505 2.505 0 0 0-1.911-.004L9.231 8.549a2.495 2.495 0 0 0-2.474-2.125Z" style="fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.0690307"/><path d="M6.757 6.217a2.703 2.703 0 0 0-2.707 2.69c0 .803.364 1.52.927 2.013v2.215a2.67 2.67 0 0 0-.927 2.013 2.703 2.703 0 0 0 2.707 2.69c1.492 0 2.71-1.207 2.71-2.69 0-.901-.453-1.698-1.14-2.185v-1.278l1.049.654a2.702 2.702 0 0 0 2.681 2.36 2.7 2.7 0 0 0 2.68-2.351l1.016-.632v1.268a2.683 2.683 0 0 0-1.105 2.164c0 1.483 1.218 2.69 2.71 2.69a2.703 2.703 0 0 0 2.707-2.69c0-.821-.38-1.552-.964-2.045V10.95a2.67 2.67 0 0 0 .964-2.043 2.703 2.703 0 0 0-2.707-2.69A2.7 2.7 0 0 0 14.7 8.428l-1.71 1.066a2.693 2.693 0 0 0-.933-.175c-.325 0-.632.066-.92.17L9.412 8.417a2.7 2.7 0 0 0-2.656-2.2Zm0 .414c1.157 0 2.11.847 2.27 1.948.008.06.043.113.094.145l1.875 1.165a.207.207 0 0 0 .188.017 2.302 2.302 0 0 1 1.753.004.207.207 0 0 0 .19-.017l1.864-1.159a.207.207 0 0 0 .095-.146 2.286 2.286 0 0 1 2.272-1.957 2.282 2.282 0 0 1 2.293 2.276c0 .73-.347 1.375-.885 1.793-.05.039-.08.1-.08.164v2.325c0 .064.03.125.08.164.538.417.885 1.065.885 1.795a2.282 2.282 0 0 1-2.293 2.276 2.284 2.284 0 0 1-2.296-2.276c0-.787.4-1.48 1.012-1.89a.207.207 0 0 0 .093-.172v-1.741a.207.207 0 0 0-.317-.175l-1.411.876a.207.207 0 0 0-.097.157 2.284 2.284 0 0 1-2.285 2.082 2.286 2.286 0 0 1-2.286-2.09.207.207 0 0 0-.097-.158L8.23 11.14a.207.207 0 0 0-.317.174v1.752c0 .07.036.135.095.173a2.273 2.273 0 0 1 1.044 1.909 2.284 2.284 0 0 1-2.295 2.276 2.282 2.282 0 0 1-2.293-2.276c0-.715.33-1.35.85-1.767a.207.207 0 0 0 .078-.162v-2.383a.207.207 0 0 0-.078-.162 2.26 2.26 0 0 1-.85-1.767 2.282 2.282 0 0 1 2.293-2.276z" style="opacity:0.99;fill:#000000;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:0.0690307"/></svg>
```

Comment:
You need to merge and join all paths before optimizing so that the resulting SVG matches the guidelines.